### PR TITLE
fix: re-Add the "Edit On GitHub"

### PR DESCRIPTION
**Pull request by ghost**
"Edit on GitHub" was removed at #4725, and this is the fix for it.

### CODE REVIEW

File: layouts/partials/footer.hbs [hunk 1]
```diff
@@ -5,7 +5,19 @@
...
+        <a href="https://openjsf.org">The OpenJS Foundation</a> | 
+        <a href="https://terms-of-use.openjsf.org">Terms of Use</a> | 
+        <a href="https://privacy-policy.openjsf.org">Privacy Policy</a> | 
+        <a href="https://bylaws.openjsf.org">Bylaws</a> | 
+        <a href="https://code-of-conduct.openjsf.org">Code of Conduct</a> | 
+        <a href="https://trademark-policy.openjsf.org">Trademark Policy</a> | 
+        <a href="https://trademark-list.openjsf.org">Trademark List</a> | 
+        <a href="https://www.linuxfoundation.org/cookies">Cookie Policy</a>
+        </p>
+        <div style="display:inline;font-size: 15px;">
```
Comment (osk2) [inline layouts/partials/footer.hbs]: We should avoid using inline styles as it's hard to maintain.

**Review by aymen94 — APPROVED** [review]

File: layouts/partials/footer.hbs (hunk 1 above)
Comment (ghost) [reply layouts/partials/footer.hbs]: @osk2 : OK it's fixed.

**Review by osk2 — APPROVED** [review]
👍🏼

**Comment by ghost** [comment]
Thanks to @aymen94 & @osk2 's reviews.

**Pull request merged**